frontend/app/components/routing/wp-show/wp-show.controller.ts

```diff
@@ -69,6 +71,11 @@ export class WorkPackageShowController extends WorkPackageViewController {
     // initialization
     this.initializeAllowedActions();
     this.setWorkPackageScopeProperties(this.workPackage);
+
+    // Clear the list checksum if we're doing something that requires reloading the list
+    this.$scope.$root.$on('workPackagesRefreshRequired', () => {
+      this.wpListChecksumService.clear();
```

Comment:
Didn't check but woudn't it make sense to place it in the wp-list.service together with the other listener for the `workPackagesRefreshRequired` event?